### DISCUSSION TITLE
Put up a list of awesome metabolic engineering tools somewhere?

### DISCUSSION THREAD

**Discussion by bdelepine** [opencobra/cobrapy#1089] [General]
Hello everyone!

I was thinking about last COBRApy meeting and more specifically about how difficult it is to onboard people to use the right computational metabolic engineering tools (master students that end up asking the same questions on the Google Group, but also wet-lab researchers that don't know which tools to use, etc.). I wonder if we could improve that by putting up something like an [awesome list](https://github.com/sindresorhus/awesome/blob/main/awesome.md) of metabolic engineering resources. We could refer to this list in COBRAPy doc and README in an attempt to redirect the exploration of new users. 

Of course, we would need to discuss the scope of the list (only Python? only GEM-related? only those maintained en the last X years? etc.).

Below a quick showcase of what I have in mind:

**Learning resources** 
* [Palsson lectures](https://www.youtube.com/watch?v=nHkYlEWNxeM&list=PLzm7DNlS3_0hryL9WsVbWV_EkwBJNt9s3) on systems biology
* [Metabolic modeling for cell factory engineering](https://biosustain.github.io/cell-factory-design-course/)

**Genome-scale models repositories**
* [BiGG Models](http://bigg.ucsd.edu/)

**WebGUI to reconstruct GEM**
* [MetaNetX](https://www.metanetx.org/)
* [ModelSEED](https://modelseed.org/)
* [MetExplore](https://vm-metexplore-prod.toulouse.inra.fr/index.html/)

**Metabolic network visualisation tools**
* [Escher](https://escher.github.io)
* [MetExploreViz](https://metexplore.toulouse.inrae.fr/metexploreViz/doc/documentation.php)
* [Cytoscape](https://cytoscape.org/) with [cy3sbml app](http://apps.cytoscape.org/apps/cy3sbml)

**GEM simulation software**
* COBRA for MATLAB, [COBRApy](https://github.com/opencobra/cobrapy/)
* [CAMEO](https://github.com/biosustain/cameo/)
* [OptFlux](http://www.optflux.org/)
* [Caffeine](https://caffeine.dd-decaf.eu/)

**Yet another category**
* More items

What do you think? Would you expect it to be useful for newcomers?

**Comment by Midnighter** [comment]
I don't have the bandwidth right now to add to your list but in general I think it's a great idea. As you mention, it will help to set the scope of the list early on.
